python-sdk/docs/CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Changelog
 
+## 1.3.1
+
+### Feature:
+- Introduce a parameter `if_exists` which user can set to `append` or `replace` for `aql.dataframe()` operator. `append` insert into an existing table (`output_table`) from a Dataframe. [#1379](https://github.com/astronomer/astro-sdk/issues/1379)
+- Temporary Schemas accounts for Location/Region when creating BigQuery schema. User can now create schema in Bigquery as per region like - `<dataset_id>__<region_id>`. [#1369](https://github.com/astronomer/astro-sdk/issues/1369)
```

Comment:
Would have been good to add a quick example. Have we covered this case in docs too? I don't see anything in https://github.com/astronomer/astro-sdk/pull/1416/files . How would a user know about this feature?

cc @phanikumv @utkarsharma2